### PR TITLE
make it compatible for android gradle build tools 1.1.0

### PR DESCRIPTION
**Pull request by nenick**
Looks like that was the only necessary change to make this plugin compatible. I still need your plugin ;)

### CODE REVIEW

**Comment by ouchadam** [comment]
seems legit thanks! I'll try and do a release tonight

**Comment by nenick** [comment]
Awesome!

File: buildSrc/build.gradle [hunk 1]
```diff
@@ -3,6 +3,6 @@ repositories {
 }
 
 rootProject.dependencies {
-    compile 'com.android.tools.build:gradle:0.14.1'
+    compile 'com.android.tools.build:gradle:1.1.0'
```
Comment (blundell) [inline buildSrc/build.gradle]: :+1:

**Comment by ouchadam** [comment]
@nenick released ! https://bintray.com/novoda/maven/gradle-android-test-plugin/view

thanks for your PR :smile_cat:

**Comment by nenick** [comment]
I don't know what I do wrong because my project can't find the 0.10.3 release but the 0.10.2

**Comment by blundell** [comment]
it might not have propogated that fast, try now! It is there:

https://bintray.com/novoda/maven/gradle-android-test-plugin/0.10.3/view

**Comment by nenick** [comment]
Looks like it is not ready yet. I will try it again at evening. https://travis-ci.org/nenick/android-gradle-template

**Comment by nenick** [comment]
Same status ... do you have the same issue?

    > Could not resolve all dependencies for configuration ':AppComponentTests:classpath'.
        > Could not find com.novoda:gradle-android-test-plugin:0.10.3.
         Searched in the following locations:
         https://jcenter.bintray.com/com/novoda/gradle-android-test-plugin/0.10.3/gradle-android-test-plugin-0.10.3.pom
         https://jcenter.bintray.com/com/novoda/gradle-android-test-plugin/0.10.3/gradle-android-test-plugin-0.10.3.jar
         https://repo1.maven.org/maven2/com/novoda/gradle-android-test-plugin/0.10.3/gradle-android-test-plugin-0.10.3.pom
         https://repo1.maven.org/maven2/com/novoda/gradle-android-test-plugin/0.10.3/gradle-android-test-plugin-0.10.3.jar
         https://oss.sonatype.org/content/repositories/snapshots/com/novoda/gradle-android-test-plugin/0.10.3/gradle-android-test-plugin-0.10.3.pom
         https://oss.sonatype.org/content/repositories/snapshots/com/novoda/gradle-android-test-plugin/0.10.3/gradle-android-test-plugin-0.10.3.jar
         file:/home/travis/.m2/repository/com/novoda/gradle-android-test-plugin/0.10.3/gradle-android-test-plugin-0.10.3.pom
         file:/home/travis/.m2/repository/com/novoda/gradle-android-test-plugin/0.10.3/gradle-android-test-plugin-0.10.3.jar
     Required by:
         com.example:AppComponentTests:unspecified

**Comment by ouchadam** [comment]
yes it's also not working for me, quite odd. It is usually instant deployment to jcenter. The api is giving 
```json
{
  "errors" : [ {
    "status" : 404,
    "message" : "Could not find resource"
  } ]
}
```
yet the bintray repository is linked to `jcenter`.

for now
```groovy
repositories {
  maven {
    url 'http://dl.bintray.com/novoda/maven/'
  }
}
```

will work to find the dependency until jcenter has properly synced

**Comment by nenick** [comment]
I got it at the same time :)

**Comment by nenick** [comment]
i have another issue now, please look at https://github.com/novoda/gradle-android-test-plugin/pull/32

**Pull request merged**